### PR TITLE
Allow subscribers to edit their own settings, enable bluelink

### PR DESCRIPTION
**Pull request by nickcatal**
This is a simple way to expose to subscribers their own settings inside the subscriber admin.

This also allows users to opt-in to enable a sync to [Bluelink](https://bluelink.org/).

### CODE REVIEW

File: app/multi_tenant/forms.py [hunk 1]
```diff
@@ -14,6 +15,53 @@ class ChangeSubscriberManageForm(forms.Form):
...
+        }
+        help_texts = {
+            "name": "The name your organization will be referred to as throughout the tools.",
+            "url": "The URL of your homepage.",
+            "privacy_policy": "The URL of your privacy policy. This is highly recommended.",
+            "sms_enabled": "If your organization has a SMS program.",
+            "sms_checkbox": "If your organization requires a checkbox to be checked by users. Leave unchecked if you have no SMS program.",
+            "sms_checkbox_default": "If, by default, your organization's SMS checkbox is checked. Leave this unchecked if you have no SMS program or no SMS checkbox.",
+            "sms_disclaimer": "If you wish to have a custom disclaimer, or next to your checkbox if you have one.",
+            "sync_tmc": mark_safe(
```
Comment (benweissmann) [inline app/multi_tenant/forms.py:49]: Is there any way to show a message if this or the bluelink integration is enabled, like "You have enabled (TMC/Bluelink) sync. Please contact your (TMC/Bluelink) support contact and tell them your VoteAmerica subscriber id is `foobar` and they will get you set up."?

Comment (benweissmann) [reply app/multi_tenant/forms.py:49]: Maybe just add this as a conditional paragraph at the bottom of the Jinja template?

File: app/multi_tenant/forms.py [hunk 2]
```diff
@@ -14,6 +15,53 @@ class ChangeSubscriberManageForm(forms.Form):
...
+                "If your data should be included in a sync to <a href='https://movementcooperative.org/'>The Movement Cooperative</a>."
```
Comment (benweissmann) [inline app/multi_tenant/forms.py:50]: This should have `target="_blank" rel="noopener noreferrer"`

File: app/multi_tenant/forms.py [hunk 3]
```diff
@@ -14,6 +15,53 @@ class ChangeSubscriberManageForm(forms.Form):
...
+            ),
+            "sync_bluelink": mark_safe(
+                "If your data should be included in a sync to <a href='https://bluelink.org/'>Bluelink</a>."
```
Comment (benweissmann) [inline app/multi_tenant/forms.py:53]: Ditto on this link

File: app/multi_tenant/forms.py (hunk 1 above)
Comment (nickcatal) [reply app/multi_tenant/forms.py:49]: Done

**Pull request merged**